en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md

```diff
@@ -1,206 +1,162 @@
-# Setting up Oracle RAC
+# Changing to Oracle RAC
 
-The following sections describe how to set up Oracle RAC to replace the default H2 database in your WSO2 product:
+By default, WSO2 API Manager uses the embedded H2 database as the database for storing user management and registry data. Given below are the steps you need to follow in order to use Oracle RAC for this purpose.
 
--   [Setting up the database and users](#SettingupOracleRAC-Settingupthedatabaseandusers)
--   [Setting up the JDBC driver](#SettingupOracleRAC-SettinguptheJDBCdriver)
+## Setting up Oracle RAC
 
-Oracle Real Application Clusters (RAC) is an option that facilitates clustering and high availability in Oracle database environments. In the Oracle RAC environment, some of the commands used in `oracle.sql` are considered inefficient. Therefore, the product has a separate SQL script ( `oracle_rac.sql` ) for Oracle RAC. The Oracle RAC-friendly script is located in the `dbscripts` folder together with other `.sql` scripts.
+The following sections describe how to set up Oracle RAC to replace the default H2 database in your WSO2 product:
 
-!!! info
-To test products on Oracle RAC, rename `oracle_rac.sql` to `oracle.sql` before running `-Dsetup` .
+-   [Setting up the database and users](#setting-up-the-database-and-users)
+-   [Setting up the drivers](#setting-up-the-drivers)
+-   [Executing db scripts to create tables on Oracle RAC database](#executing-db-scripts-to-create-tables-on-oracle-rac-database)
 
+Oracle Real Application Clusters (RAC) is an option that facilitates clustering and high availability in Oracle database environments. In the Oracle RAC environment, some of the commands used in `oracle.sql` are considered inefficient. Therefore, the product has a separate SQL script, `oracle_rac.sql` for Oracle RAC. The Oracle RAC-friendly script is located in the `dbscripts` folder together with other `.sql` scripts.
```

Comment:
Shall we change the path as `<API-M_HOME>/dbscripts`?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/619/commits/2c49dea90e2c24ab57b4dbf7c8549ff78be98a05